<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="1" tests="6" time="24.875">
 <testsuite name="Calculator Test" timestamp="2020-01-24T11:45:02" hostname="localhost" time="24.875" errors="0" tests="6" skipped="0" disabled="0" failures="1">
  <testcase classname="Calculator Test" name="Verify application url" time="2.704" />
  <testcase classname="Calculator Test" name="Add two numbers" time="4.476" />
  <testcase classname="Calculator Test" name="Substract two numbers" time="4.488" />
  <testcase classname="Calculator Test" name="Multiply two numbers" time="4.512" />
  <testcase classname="Calculator Test" name="Devide two numbers" time="4.258" />
  <testcase classname="Calculator Test" name="Modulo % two numbers" time="4.436">
   <failure type="toEqual" message="Expected &apos;2&apos; to equal &apos;200&apos;."><![CDATA[Error: Failed expectation
    at UserContext.<anonymous> (/Users/manasp/Documents/ProtractorTest/outputjs/specs/calculator.js:64:83)
    at /Users/manasp/Documents/ProtractorTest/node_modules/jasminewd2/index.js:112:25
    at new ManagedPromise (/Users/manasp/Documents/ProtractorTest/node_modules/selenium-webdriver/lib/promise.js:1077:7)
    at ControlFlow.promise (/Users/manasp/Documents/ProtractorTest/node_modules/selenium-webdriver/lib/promise.js:2505:12)
    at schedulerExecute (/Users/manasp/Documents/ProtractorTest/node_modules/jasminewd2/index.js:95:18)
    at TaskQueue.execute_ (/Users/manasp/Documents/ProtractorTest/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/Users/manasp/Documents/ProtractorTest/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at /Users/manasp/Documents/ProtractorTest/node_modules/selenium-webdriver/lib/promise.js:2974:25
    at /Users/manasp/Documents/ProtractorTest/node_modules/selenium-webdriver/lib/promise.js:668:7]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>